pkg/controller/operators/olm/apiservices.go

```diff
@@ -298,29 +298,30 @@ func apiServiceDescriptionsForDeployment(descs []v1alpha1.APIServiceDescription,
 	return result
 }
 
-func (a *Operator) installOwnedAPIServiceRequirements(csv *v1alpha1.ClusterServiceVersion, strategy install.Strategy) (install.Strategy, error) {
+func (a *Operator) installOwnedAPIServiceRequirements(csv *v1alpha1.ClusterServiceVersion, strategy install.Strategy) (install.Strategy, map[string][]byte, error) {
 	logger := log.WithFields(log.Fields{
 		"csv":       csv.GetName(),
 		"namespace": csv.GetNamespace(),
 	})
 
 	// Assume the strategy is for a deployment
+	deploymentNameToCAPEM := map[string][]byte{}
 	strategyDetailsDeployment, ok := strategy.(*v1alpha1.StrategyDetailsDeployment)
 	if !ok {
-		return nil, fmt.Errorf("unsupported InstallStrategy type")
+		return nil, deploymentNameToCAPEM, fmt.Errorf("unsupported InstallStrategy type")
```

Comment:
wait could we return a `nil` map here instead?

Reply:
Actually, we can just return nil because maps are pointers, as shown [here](https://dave.cheney.net/2017/04/30/if-a-map-isnt-a-reference-variable-what-is-it).